### PR TITLE
Add aarch64 support.

### PR DESCRIPTION
**Pull request by tdaede**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -37,7 +37,8 @@ RUN \
 		gettext-base \
 		git-core \
 		gpg \
-		gpg-agent \
+     	gpg-agent \
```
Comment (tmatth) [inline Dockerfile:40]: Weird indentation here?

File: awcy_server.ts [hunk 2]
```diff
@@ -122,6 +122,7 @@ function process_build_queue() {
     env['EXTRA_OPTIONS'] = build_job.extra_options;
     env['BUILD_OPTIONS'] = build_job.build_options;
     env['RUN_ID'] = build_job.run_id;
+    env['ARCH'] = build_job.arch ? build_job.arch : 'x86_64';
```
Comment (tmatth) [inline awcy_server.ts:125]: Idem

File: build_codec.sh [hunk 3]
```diff
@@ -85,19 +85,23 @@ case "${CODEC}" in
...
+      ARCH_OPTIONS=""
+    fi
+    cmake ../ -DENABLE_TESTS=0 -DENABLE_DOCS=0 -DCMAKE_BUILD_TYPE=Release ${ARCH_OPTIONS} ${BUILD_OPTIONS}
     make -j$(nproc)
     popd
-    mkdir -p x86_64
-    mv cmake-build/aomenc cmake-build/aomdec x86_64/
+    mkdir -p "$ARCH"
+    mv cmake-build/aomenc cmake-build/aomdec ./
     echo "-- Finished x86_64 Build --"
```
Comment (tmatth) [inline build_codec.sh:104]: Does this need to be replaced with ARCH as well?

**Review by tmatth — APPROVED** [review]
Lgtm with nits addressed, I can't tell if the patch is doing weird things with indentation or it's just being rendered that way on my phone

File: awcy_server.ts (hunk 2 above)
Comment (tdaede) [reply awcy_server.ts:125]: I don't think it is?

File: www/src/components/SubmitJobForm.tsx [hunk 4]
```diff
@@ -187,7 +201,7 @@ export class SubmitJobFormComponent extends React.Component<{
       </FormGroup>
 
       <FormGroup validationState={this.getValidationState("build")}>
-      <ControlLabel>Extra Build Options (experiments)</ControlLabel>
+        <ControlLabel>Extra Build Options (experiments)</ControlLabel>
```
Comment (tmatth) [inline www/src/components/SubmitJobForm.tsx:204]: nit: extra whitespace

File: www/src/components/SubmitJobForm.tsx [hunk 5]
```diff
@@ -167,7 +181,7 @@ export class SubmitJobFormComponent extends React.Component<{
 
       <FormGroup validationState={this.getValidationState("commit")}>
         <FormControl type="text" placeholder="Git Commit Reference (Hash/Branch/Tag)"
-          value={job.commit} onChange={this.onInputChange.bind(this, "commit")} />
+           value={job.commit} onChange={this.onInputChange.bind(this, "commit")} />
```
Comment (tmatth) [inline www/src/components/SubmitJobForm.tsx:184]: nit: extra whitespace

**Review by tmatth — APPROVED** [review]

**Pull request merged**